Makefile

```diff
@@ -1,11 +1,11 @@
 
 install:
-	virtualenv .
+	python2.7 -m virtualenv .
```

Comment:
I don't remember exactly, but https://www.python.org/dev/peps/pep-0338/ suggests that `python -m` has been around for a while.